ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -8,6 +8,10 @@ declaration template components/ssh/schema;
 include 'quattor/types/component';
 include 'pan/types';
 
+variable SSH_SCHEMA_VERSION ?= '5.3';
```

Comment:
I was reluctant to add logic to the component as there doesn't seem to be much in there at the moment, I also felt it better to hit an error earlier at compilation rather than at configuration. This is the only change I've come across changing from RHEL6 to RHEL7